config/routes.rb

```diff
@@ -27,5 +27,18 @@
     get 'unlink', on: :member
   end
 
+  resources :teams do
+    member do
+      post :assign_ownership
```

Comment:
Consider using a different route definition, e.g. `post 'teams/assign_ownership/:member_id' => 'teams#assign_ownership'`

Reply:
might wanna have a look at this together with -- since this is IMO not (easily) possible. we are using GET parameters intentionally since the id in the url is used for the team and not for the member -- also GET parameters are easily accessed within Abilities.
will look into it, tho